math/mathcore/inc/Fit/FitExecutionPolicy.h

```diff
@@ -2,7 +2,7 @@
 #define ROOT_Fit_FitExecutionPolicy
 namespace ROOT{
    namespace Fit{
-      enum ExecutionPolicy { kSerial, kMultithread, kMultiprocess };
+      enum class ExecutionPolicy { kSerial, kMultithread, kMultiprocess };
```

Comment:
I wonder if we shouldn't be using the type itself rather than its value to distinguish between the policies.
Using the type resolution has no runtime cost, while the value has to be passed (and probably checked).